scaleway/resource_lb_lb_beta.go

```diff
@@ -0,0 +1,189 @@
+package scaleway
+
+import (
+	"github.com/hashicorp/terraform/helper/schema"
+	lb "github.com/scaleway/scaleway-sdk-go/api/lb/v1"
+)
+
+func resourceScalewayLbLbBeta() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceScalewayLbLbBetaCreate,
+		Read:   resourceScalewayLbLbBetaRead,
+		Update: resourceScalewayLbLbBetaUpdate,
+		Delete: resourceScalewayLbLbBetaDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+		SchemaVersion: 0,
+		Timeouts: &schema.ResourceTimeout{
+			Create: &BaremetalServerResourceTimeout,
+			Delete: &BaremetalServerResourceTimeout,
+		},
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				Description: "Name of the lb",
+			},
+			"type": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "The type of load-balancer you want to create",
+			},
+			"tags": {
+				Type: schema.TypeList,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+				Optional:    true,
+				Description: "Array of tags to associate with the load-balancer",
+			},
+			"ips": {
```

Comment:
IP is computed for now when the will allow IP previsioning before we will have a instance style attach/detach paradigm. As this will break soon i remove list for now